__sandbox__/Sandbox.tsx

```diff
@@ -81,17 +81,19 @@ export function Sandbox(): any {
       return a * b * 4
     })
 
-    const session = await flatfile.startOrResumeImportSession()
-    session.on('init', ({ workspaceId, batchId }) => {
```

Comment:
This was working only because of the setTimeout in the ImportSession constructor. After removing that for keeping only one emit of init event we needed to pass this as a callback.